app/frontend/courses/students/components/CoursesStudents__StudentOverlay.res

```diff
@@ -246,6 +246,50 @@ let quizPerformanceChart = (averageQuizScore, quizzesAttempted) =>
   | None => React.null
   }
 
+let milestoneTargetsCompletionStats = studentDetails => {
+  let milestoneTargets = studentDetails->StudentDetails.milestoneTargetsCompletionStatus
+
+  let totalMilestoneTargets = Js.Array2.length(milestoneTargets)
+
+  let completedMilestoneTargets =
+    milestoneTargets->Js.Array2.filter(target => target.completed == true)->Js.Array2.length
+
+  let milestoneTargetCompletionPercentage = string_of_int(
+    int_of_float(
+      float_of_int(completedMilestoneTargets) /. float_of_int(totalMilestoneTargets) *. 100.0,
+    ),
+  )
+
+  <div className="flex items-center space-x-4">
+    <p className="text-xs font-medium text-gray-500 flex whitespace-nowrap">
+      {(completedMilestoneTargets->string_of_int ++ " / " ++ totalMilestoneTargets->string_of_int)
+        ->str}
+    </p>
+    <div className="flex items-center space-x-1">
```

Comment:
@vinutv 

This one as well for mobiles and tablets?

![image](https://github.com/pupilfirst/pupilfirst/assets/53794102/40ddbba7-d90a-4aed-bb2f-77c5a6ceb74e)